data/src/main/java/me/androidbox/data/remote/authentication/AuthenticationRepositoryImp.kt

```diff
@@ -47,7 +47,7 @@ class AuthenticationRepositoryImp @Inject constructor(
         /* TODO Is this the best way to extract the value from the Result<T> */
         val loginDto = result.getOrNull()
         return if(loginDto != null) {
-            val login = Login(
+            val login = LoginUser(
```

Comment:
How about `LoggedInUser` or `AuthenticatedUser`? That's how I called it, but up to you ofc

Reply:
@philipplackner I went with the name `AuthenticatedUser` :blush: